docs/docs/dcap.md

```diff
@@ -1,96 +1,70 @@
 ## The DCAP protocol
 
-There are two protocols for establishing what code runs in an enclave: EPID and DCAP. EPID is an older
-protocol designed for consumer applications and includes some sophisticated privacy features. For servers
-where the IP address doesn't need to be hidden (because it's public in DNS, to begin with), these features
-aren't helpful, and thus there is DCAP (_Data Center Attestation Primitives_). DCAP requires more modern hardware
-but is otherwise simpler and more robust. You may also see DCAP referred to as "ECDSA attestation".
+Intel SGX uses the Data Center Attestation Primitives (DCAP) protocol to prove what code runs in an enclave. DCAP is 
+also known as ECDSA attestation.
 
-!!!Important
+In DCAP, repeated attestation requests are served from a cache rather than forwarded to Intel. A newly installed
+machine obtains a machine certificate from Intel via the cache, which may then be persisted to disk. All this is
+automated for you.
 
-    Intel does not provide EPID attestation support on any Xeon Scalable processors including the Ice Lake Xeon CPUs 
-    and any future generations. Please ensure you are using DCAP attestation on these platforms.
+## DCAP Client
 
+To perform attestation using DCAP, Conclave needs a way to gather information about the platform the enclave is
+hosted on. This information provides proof from Intel that the system supports SGX and that it is patched and up to
+date. A DCAP client is used for this function. It is basically a library named `libdcap_quoteprov.so.1` with a 
+symbolic link at `libdcap_quoteprov.so`, which is to be installed in `/usr/lib/` or `/usr/lib/x86_64-linux-gnu`.
 
-In DCAP, repeated attestation requests aren't forwarded to Intel, but rather served from a cache. A newly installed
-machine obtains a machine certificate from Intel via the cache, which may then be persisted to disk. All this is
-automated for you.
+You can use one of the three available DCAP clients listed below:
 
-As cloud providers run caches, DCAP supports vendor-specific plugins. Intel provides a default one
-which requires a [subscription](https://api.portal.trustedservices.intel.com/provisioning-certification).
+1. Intel DCAP client
+2. Azure DCAP client
+3. Conclave-Azure bundled client
 
-Azure provides a DCAP [plugin](https://github.com/microsoft/Azure-DCAP-Client) that does not require a subscription. Conclave
-bundles and uses Azure's DCAP plugin by default. The Azure caches are open to the public internet and can be used from anywhere. Azure Confidential Computing instances like DC4s_v3 come pre-provisioned for DCAP and as Conclave
-comes with the necessary libraries bundled, you don't need to do any further setup.
+To avoid conflicts between DCAP client plugins, you must uninstall the plugins that you don't require.
```

Comment:
This is the first time we use the word `plugin` in this doc. I would keep things consistent and redundant and use client instead.
I would say something like
`To avoid conflicts between DCAP clients, you must uninstall the ones you don't require`

Or in the sentence above, we should clarify "A DCAP client (also called **DCAP plugin**)" is used for this function`